src/controllers/renewal-registration.js

```diff
@@ -1,7 +1,36 @@
 import {ReturnState} from './_base.js';
 
-const renewalRegistrationController = (_request) => {
-  // Proceed to the next page.
+// Cleans input checking if its undefined and trimming white space from the text
+const cleanInput = (body) => {
+  return {
+    renewalRegistrationNumber:
+      body.renewalRegistrationNumber === undefined ? undefined : body.renewalRegistrationNumber.trim()
+  };
+};
+
```

Comment:
I think this should be 
body?.renewalRegistrationNumber === undefined ? undefined : body.renewalRegistrationNumber.trim()

ie have a question mark after body to allow for [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)